Libplanet.Stun/Stun/TurnClient.cs

```diff
@@ -267,11 +267,35 @@ public void Dispose()
         private async Task ProcessMessage()
         {
             NetworkStream stream = _control.GetStream();
+            int retryCount = 3;
+
             while (_control.Connected)
             {
                 try
                 {
-                    StunMessage message = await StunMessage.Parse(stream);
+                    StunMessage message;
+                    try
+                    {
+                        message = await StunMessage.Parse(stream);
+                    }
+                    catch (TurnClientException e)
+                    {
+                        if (retryCount > 0)
+                        {
+                            retryCount -= 1;
+                            await Task.Delay(1000);
+                            continue;
+                        }
+
+                        Log.Error("Failed to parse StunMessage. {e}", e);
+                        foreach (var tcs in _responses.Values)
+                        {
+                            tcs.SetCanceled();
```

Comment:
Is it okay we assume it can be cancelable always?.

```suggestion
                            tcs.TrySetCanceled();
```